package.json

```diff
@@ -56,7 +56,7 @@
     "react-transform-hmr": "^1.0.4",
     "style-loader": "^0.13.1",
     "url-loader": "^0.5.7",
-    "webpack": "^1.14.0",
+    "webpack": "2",
```

Comment:
Pre-yarn I would have agreed, but because versions are locked in `yarn.lock` and yarn will never downgrade a package unless told so, this is probably the most readable/simplest setup? 2 means any "2.x" version which sounds good to me.

We could pick `^2.0.0` which means "any 2.x.x version greater than or equal to 2.0.0" and have a different version in yarn.lock as well… but it'd be exactly the same no?